### PR TITLE
show toast msg when no logged in

### PR DESCRIPTION
**Pull request by jiabaow**

### CODE REVIEW

File: app/src/androidTest/java/ch/sdp/vibester/fragment/SearchUserFragmentNotLoggedInTest.kt [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+
+
+    @BindValue @JvmField
+    val mockUsersRepo = mockk<DataGetter>()
+
+    private fun createMockInvocation() {
+        val mockUser1 = User("mockUser1", uid = "mockUser1uid")
+        val mockUser2 = User("mockUser2", uid = "mockUser2uid")
+        val mockUser3 = User("mockUser3", uid = "mockUser3uid")
+        val mockUser = User("mockUser", uid = "mockUseruid", following = mapOf(Pair(mockUser2.uid,true), Pair(mockUser3.uid,true)))
```
Comment (MaximeZmt) [inline app/src/androidTest/java/ch/sdp/vibester/fragment/SearchUserFragmentNotLoggedInTest.kt:47]:
```suggestion
        val mockUser = User("mockUser", uid = "mockUseruid", following = mapOf(Pair(mockUser2.uid, true), Pair(mockUser3.uid, true)))
```

File: app/src/androidTest/java/ch/sdp/vibester/fragment/SearchUserFragmentNotLoggedInTest.kt [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+
+        val mockUIDs = arrayListOf("mockUser1uid","mockUser2uid","mockUser3uid")
```
Comment (MaximeZmt) [inline app/src/androidTest/java/ch/sdp/vibester/fragment/SearchUserFragmentNotLoggedInTest.kt:49]:
```suggestion
        val mockUIDs = arrayListOf("mockUser1uid", "mockUser2uid", "mockUser3uid")
```

**Review by MaximeZmt — COMMENTED** [review]
nice work, LGTM 👍  two minor things below

**Review by MaximeZmt — APPROVED** [review]
👍

File: app/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -196,5 +196,7 @@
 
     <string name="PartyRoom_joinedPeople">Joined People:</string>
 
+    <string name="searchUser_pleaseLoggedInFirst">Please logged in first</string>
```
Comment (kamilababayeva) [inline app/src/main/res/values/strings.xml:199]:
```suggestion
    <string name="searchUser_pleaseLogInFirst">Please login first</string>
```

**Review by kamilababayeva — APPROVED** [review]
Loooks good! I have changed the string, let me know if you agree!

**Pull request merged**